demisto_sdk/__main__.py

```diff
@@ -153,6 +157,15 @@ def main(config, version):
         if last_release and cur_version != last_release:
             print_warning(f'however version {last_release} is available.\n'
                           f'You should consider upgrading via "pip3 install --upgrade demisto-sdk" command.')
+        if release_notes:
+            rn_entries = tools.get_release_note_entries(cur_version)
```

Comment:
import it straight from tools. no need for `tools.YYYY`

Reply:
done.